webapp/packages/plugin-navigation-tree/src/NavigationTree/ElementsTree/useElementsTree.ts

```diff
@@ -170,98 +169,96 @@ export function useElementsTree(options: IOptions): IElementsTree {
       return false;
     }
   }
+  const [loadingNodes] = useState(() => observable.map(new Map<string, Promise<void>>(), { deep: false }));
```

Comment:
I think we should move it to `elementsTree` state as `observable.shallow` to keep all trees tools in one place